app/registries/menu/toggle.rb

```diff
@@ -17,7 +17,7 @@ def children_hash
         index = list.index(child)
         child.is_a?(Menu::Divider) && (index == list.size - 1 || list[index + 1].is_a?(Menu::Divider))
       end
-      list.map(&:to_hash)
+      list.map(&:to_hash).compact
```

Comment:
The conditional was causing `nil` to make its way into the children. That was causing the entire website frame to fail rendering.